webapp/channels/src/components/channel_selector_modal/channel_selector_modal.tsx

```diff
@@ -4,13 +4,12 @@
 import React from 'react';
 import {Modal} from 'react-bootstrap';
 import type {IntlShape} from 'react-intl';
-import {injectIntl, FormattedMessage} from 'react-intl';
+import {FormattedMessage, injectIntl} from 'react-intl';
```

Comment:
Why did we change the order here?

Reply:
![image](https://github.com/user-attachments/assets/debe8a41-cc01-41a5-a581-7c617685a2bd)

I thought that's the pattern you're following.. Loll, it's not same everywhere!

Reply:
Let me know if I have to reverse it, after this I'll squash all the commits into one!

Reply:
Yes we recommend not introducing any unintended changes and keeping the scope to what's required in the PR itself. Not saying this is a big change but it's a good practice to follow

Reply:
Done!!